doc/en/user/source/production/container.rst

```diff
@@ -54,6 +52,17 @@ For more information about JVM configuration, see the article `Performance tunin
     
    The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
    
+Enable the Marlin rasterizer
+----------------------------
+
+The Marlin rasterizer in Java 8 and getting better performance and scalability while rendering vector data in Java 8. 
+In order to enable it add the following among the JVM startup options::
+
+     -Xbootclasspath/a:$MARLIN_JAR 
+     -Dsun.java2d.renderer=org.marlin.pisces.MarlinRenderingEngine 
+
+where ``$MARLIN_JAR`` is the location of the ``marlin*.jar`` file located in the geoserver/WEB-INF/lib directory.
```

Comment:
Meh... the best position for the Marlin jar is inside the JDK, jre/lib/ext.
By putting it in jetty/lib you're kicking out of the game the .war deployers, which will have to download Marlin separately, and it will be harder to place it there than to place it in the war.
The war position is still problematic, because some containers do not explode the .war.

Anyways, if you feel strongly, go for it, fork this pull request and do the work. I asked for help because I won't be able to add any further help on this one.